src/benchmarklib/benchmark_config.hpp

```diff
@@ -20,8 +20,8 @@ using TimePoint = std::chrono::steady_clock::time_point;
 class BenchmarkConfig {
  public:
   BenchmarkConfig(const BenchmarkMode init_benchmark_mode, const ChunkOffset init_chunk_size,
-                  const EncodingConfig& init_encoding_config, const bool init_indexes, const int64_t init_max_runs,
-                  const Duration& init_max_duration, const Duration& init_warmup_duration,
+                  const EncodingConfig& init_encoding_config, const bool init_chunk_indexes,
```

Comment:
Why does the benchmark config not store table index information as well?

Reply:
Same as above: I mistakenly removed it. I'll add it again.